nginx.conf

```diff
@@ -50,6 +51,30 @@ server {
       deny all;
     }
 
+    # Block all access to package and config files
+    # Note if WAF is deployed this should already be handled by WAF
+    location ^~ /package.json {
```

Comment:
Do we have apps that may be exposing `node_modules` or these sorts of files, or is this here to mimic the WAF and be extra cautious?

Reply:
I don't think these are exposed on purpose

And for WAF, I think by default our WAF is configure for only audit and log violation rather than blocking them https://github.com/uc-cdis/cloud-automation/blob/master/gen3/lib/manifestDefaults/modsec/crs-setup.conf#L96-L97 🤔 , so we better just block these from within Portal so the vulnerabilities are fixed whether WAF exists or not

Reply:
and I don't know why WAF is put in that mode by default, need to ask more ppl